docs/rhoas/rhoas-service-contexts/README.adoc

```diff
@@ -313,15 +313,15 @@ RHOAS_SERVICE_ACCOUNT_CLIENT_ID=_<client-id>_
 RHOAS_SERVICE_ACCOUNT_CLIENT_SECRET=_<client-secret>_
 RHOAS_SERVICE_ACCOUNT_OAUTH_TOKEN_URL={sso-token-url}
 ## Generated by rhoas cli
```

Comment:
@rkpattnaik780 Is this a duplicate line?

Reply:
Hi @jbyrne-redhat , this line is present in both the env files, and since we use `cat` to do the append, entire content is copied.

Reply:
Ah, makes sense. It was only a small thing, but thanks for verifying.